docs/UserGuide.adoc

```diff
@@ -145,35 +150,35 @@ Format: `find [flag] KEYWORD`
 
 Examples:
 
-* `find -g n/John Smith` +
+* `find -g John Smith` +
 Find guest(s) with "John" or "Smith" or both in their names.
-* `find -g r/85` +
-Find guest currently staying in room 85
-* `find -r 85` +
-Find room 85
+* `find -g r/085` +
+Find guest currently staying in room 085
+* `find -r 085` +
+Find room 085
 * `find -r g/John Smith` +
-Find room in which guest John Smith is currently staying
+Find room for which guest(s) with "John" or "Smith" or both in their names, has made a booking
+* `find -r double` +
+Find all double rooms
+* `find -r avail` +
+Find all available rooms today
+* `find -r avail from/09/12/18 to/10/12/18` +
+Find rooms that are available from 09/12/18 10/12/18. Input dates must be in DD/MM/YY format.
 
 === Listing all guests : `list`
 
-Shows a list of rooms or guests, depending on the input flag +
+Shows the entire list of rooms, or guests, depending on the input flag +
```

Comment:
Think the original phrasing was clearer as this implies it's (list of rooms) or (guests). Or can just be explicit and say "list of rooms or list of guests".